UPGRADE-GUIDE.md

```diff
@@ -0,0 +1,1298 @@
+- `--pf-c-alert--m-inline__icon--PaddingTop`
+- `--pf-c-alert--m-inline__icon--PaddingRight`
+- `--pf-c-alert--m-inline__icon--PaddingBottom`
+- `--pf-c-alert--m-inline__icon--PaddingLeft`
+- `--pf-c-alert--m-inline--m-warning__icon--FontSize`
+- `--pf-c-alert--m-inline--m-success__icon--Color`
+- `--pf-c-alert--m-inline--m-success--before--BackgroundColor`
+- `--pf-c-alert--m-inline--m-danger__icon--Color`
+- `--pf-c-alert--m-inline--m-danger--before--BackgroundColor`
+- `--pf-c-alert--m-inline--m-warning__icon--Color`
+- `--pf-c-alert--m-inline--m-warning--before--BackgroundColor`
+- `--pf-c-alert--m-inline--m-info__icon--Color`
+- `--pf-c-alert--m-inline--m-info--before--BackgroundColor`
+
+### Application launcher
+- Applied external link icon class to icon wrapper [(#2904)](https://github.com/patternfly/patternfly/pull/2904)
+- Removed separator in favor of divider component [(#2944)](https://github.com/patternfly/patternfly/pull/2944)
+- Removed hover, active, focus, and disabled from workspace [(#2975)](https://github.com/patternfly/patternfly/pull/2975)
+- Cleaned up variables [(#3012)](https://github.com/patternfly/patternfly/pull/3012)
+
+Updated markup:
+- Moves the class `.pf-c-app-launcher__menu-item-external-icon` from the external link icon itself to a `<span>` that wraps the external link icon. Any instances of the external link icon should be wrapped in a `<span>` and the `.pf-c-app-launcher__menu-item-external-icon` class should be moved from the icon to the `<span>`.
+
+Removed classes:
+- `.pf-c-app-launcher__separator` - element was removed. Use the divider component instead.
+- `.pf-m-hover` from `.pf-c-app-launcher__toggle` and `.pf-c-app-launcher__menu`
+- `.pf-m-focus` from `.pf-c-app-launcher__toggle` and `.pf-c-app-launcher__menu`
+- `.pf-m-disabled` from `.pf-c-app-launcher__toggle`
+  - The `:hover`, `:active`, `:focus`, and `:disabled` selectors still have styles applied to them
+
+Renamed variables:
+- `--pf-c-app-launcher--m-top__menu--Transform` to `--pf-c-app-launcher--m-top__menu--TranslateY`
+- `--pf-c-app-launcher__menu-item-external-icon--Transform` to `--pf-c-app-launcher__menu-item-external-icon--TranslateY`
+
+Removed variables:
+- `--pf-c-app-launcher__group--PaddingTop`
+- `--pf-c-app-launcher__group--first-child--PaddingTop`
+- `--pf-c-app-launcher__menu--BorderWidth`
+- `--pf-c-app-launcher__separator--Height`
+- `--pf-c-app-launcher__separator--BackgroundColor`
+- `--pf-c-app-launcher__separator--MarginTop`
+- `--pf-c-app-launcher__separator--MarginBottom`
+- `--pf-c-app-launcher__separator--Height`
+
+### Backdrop
+- Removed blur [(#3009)](https://github.com/patternfly/patternfly/pull/3009)
+
+Renamed variables:
+- `--pf-c-backdrop--Color` to `--pf-c-backdrop--BackgroundColor`
+
+Removed variables:
+- `--pf-c-backdrop--BackdropFilter`
+
+### Breadcrumb
+- Added word-break, display link and icon inline, move divider [(#2916)](https://github.com/patternfly/patternfly/pull/2916)
```

Comment:
It's just the [PR title/commit message](https://github.com/patternfly/patternfly/pull/2916). What it actually does is:

add word-break - enables long strings as breadcrumb items to break to multiple lines if needed (like a really long word)

display link and icon inline - displays the divider icon and breadcrumb item text inline with (beside) one another when breadcrumb items wrap to multiple lines as opposed to allowing the divider and breadcrumb item text to wrap independently of one another.

move divider - dividers were moved to come before the breadcrumb item link instead of after.

Also, the notes from the PR about these 3 things:

> This PR changes the visual appearance of breadcrumbs when they break to multiple lines.
> * The text will now break if there is a long string in an item link that is wider than the viewport
> * The divider is displayed before the link text, instead of after it
> * The divider will also always appear inline with the item text instead of wrapping to a new line in some cases.

Reply:
Updated to use those 3 bullet points!